docs/source/student_tutorial/index.rst

```diff
@@ -0,0 +1,48 @@
+============
+Getting Started
+============
+
+Revolve2 is a library that allows you to use evolutionary computing for modular robot evolution and simulation of those robots.
+Additionally these modular robots can be built physically and subsequently be controlled by the brain of your simulated robots.
+In this tutorial we will go over all steps for you to start researching ASAP.
+
+-------------
+Step 1: Installation
+-------------
+The first step is to install revolve2. For this follow the detailed tutorial from :ref:`Installation Guide<installation/index>`.
+Keep in mind that installation can vary depending on the release you choose.
+
+---------------------
+Step 2: Play with examples
+---------------------
+Depending on your topic of research some different functionality in revolve2 might be interesting to you.
+One good starting point to get a better understanding of the inner workings of revolve2, are the examples found in the repository.
```

Comment:
in the repository's `examples` directory. students can be more blind than you think :ok_hand: